#
# base patch: meta/recipes-core/sysvinit/sysvinit/crypt-lib.patch
# base branch: warrior
# base commit: 485b4a0e7a95a2ca0ff21637d073ee510468b9de
#

Upstream-Status: Inappropriate [configuration]

# The src Makefile was checking for libcrypt.a on the host, not in the
# build environment.  This patch checks for $LCRYPT in the environment
# and uses it if it's there.
# - jdike@linux.intel.com

Index: sysvinit-2.88dsf/src/Makefile
===================================================================
--- sysvinit-2.88dsf.orig/src/Makefile
+++ sysvinit-2.88dsf/src/Makefile
@@ -97,9 +97,13 @@ else
 endif
 
 # Additional libs for GNU libc.
+ifneq ($(LCRYPT),)
+  SULOGINLIBS	+= $(LCRYPT)
+else
 ifneq ($(wildcard /usr/lib*/libcrypt.*),)
   SULOGINLIBS	+= -lcrypt
 endif
+endif
 
 # Additional libs for GNU libc / multiarch on Debian based systems.
 ifneq ($(wildcard /usr/lib/*/libcrypt.*),)
